--- almanah-0.12.3/data/meson.build.orig	2021-03-05 16:05:46.935089000 +0100
+++ almanah-0.12.3/data/meson.build	2023-05-30 22:14:56.161300753 +0200
@@ -1,6 +1,5 @@
 desktop = 'almanah.desktop'
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop + '.in',
   output: desktop,
@@ -19,7 +18,6 @@ endif
 
 appdata = 'almanah.appdata.xml'
 i18n.merge_file(
-  appdata,
   input: appdata + '.in',
   output: appdata,
   po_dir: po_dir,
